openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/couchbaseConnection.json

```diff
@@ -1,60 +1,71 @@
+  "javaType": "org.openmetadata.schema.services.connections.database.CouchbaseConnection",
+  "definitions": {
+    "couchbaseType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": [
+        "Couchbase"
+      ],
+      "default": "Couchbase"
     },
-    "properties": {
-      "type": {
-        "title": "Service Type",
-        "description": "Service Type",
-        "$ref": "#/definitions/couchbaseType",
-        "default": "Couchbase"
-      },
-      "scheme": {
-        "title": "Connection Scheme",
-        "description": "Couchbase driver scheme options.",
-        "$ref": "#/definitions/couchbaseScheme",
-        "default": "couchbase"
-      },
-    
-        "bucket": {
-            "title": "Connection Bucket",
-            "description": "Couchbase connection Bucket options.",
-            "type": "string"
-        },
-        "username": {
-            "title": "Username",
-            "description": "Username to connect to Couchbase. This user should have privileges to read all the metadata in Couchbase.",
-            "type": "string"
-        },
-        "password": {
-            "title": "Password",
-            "description": "Password to connect to Couchbase.",
-            "type": "string",
-            "format": "password"
-        },
-        "hostport": {
-            "title": "Hostport",
-            "description": "Hostname of the Couchbase service.",
-            "type": "string"
-        }
+    "couchbaseScheme": {
+      "description": "Couchbase driver scheme options.",
+      "type": "string",
+      "enum": [
+        "couchbase"
+      ],
+      "default": "couchbase"
+    }
+  },
+  "properties": {
+    "type": {
+      "title": "Service Type",
+      "description": "Service Type",
+      "$ref": "#/definitions/couchbaseType",
+      "default": "Couchbase"
     },
-    "additionalProperties": false,
-    "required": ["hostport", "username", "password"]
+    "scheme": {
+      "title": "Connection Scheme",
+      "description": "Couchbase driver scheme options.",
+      "$ref": "#/definitions/couchbaseScheme",
+      "default": "couchbase"
+    },
+    "bucket": {
+      "title": "Connection Bucket",
+      "description": "Couchbase connection Bucket options.",
+      "type": "string"
+    },
+    "username": {
+      "title": "Username",
+      "description": "Username to connect to Couchbase. This user should have privileges to read all the metadata in Couchbase.",
+      "type": "string"
+    },
+    "password": {
+      "title": "Password",
+      "description": "Password to connect to Couchbase.",
+      "type": "string",
+      "format": "password"
+    },
+    "hostport": {
+      "title": "Hostport",
+      "description": "Hostname of the Couchbase service.",
+      "type": "string"
+    },
+    "supportsMetadataExtraction": {
+      "title": "Supports Metadata Extraction",
+      "$ref": "../connectionBasicType.json#/definitions/supportsMetadataExtraction"
+    }
```

Comment:
did the formatting in this file and added the `supportsMetadataExtraction` in couchbase db